### PR TITLE
Header section

### PR DESCRIPTION
**Pull request by Sofinaye**
With this feature I have:

- Added a toolbar

- Added a header

- Added a well-structured readme

- Added a license file 

- Added a well-functioning linter

### CODE REVIEW

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Sofinaye , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that does not have a professional readme file (You are missing the `live demo` and the `Show your support` sections), please make sure that you use the [Microverse readme template](https://github.com/microverseinc/readme-template) and customize it to match your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,120 @@
...
+  git clone git@github.com:Sofinaye/Portfo.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👤 Author <a name="authors"></a>
+  Abenezer
+
+- GitHub: [@githubhandle](https://github.com/Sofinaye)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
```
Comment (huseyinbiyik) [inline README.md:80]: Kindly add your Twitter link here but if you don't have one, you can remove this broken link. It is important not to have a broken link on your README to keep it professional 👍

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @Sofinaye 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by PinkMoon25 — APPROVED** [review]
Hi @Sofinaye , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linter Checks :heavy_check_mark: 
github flow :heavy_check_mark: 
Flexbox :heavy_check_mark: 
Follows figma design :heavy_check_mark: 
Semantics :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**